### PR TITLE
Feat/126 app role get

### PR DESCRIPTION
**Pull request by franTarkenton**
Hopefully this is better

### CODE REVIEW

File: server/backend/api/app/crud/crud_application.py [hunk 1]
```diff
@@ -92,6 +96,31 @@ def deleteFamApplication(db: Session, application_id: int):
...
+
+    application = (
+        db.query(models.FamRole)
+        # .join(models.FamRole)
+        # .options(load_only("application_id"))
+        .filter(models.FamRole.application_id == application_id,
+                models.FamRole.parent_role_id == None)
+        .all()
+    )
+    return application
```
Comment (ianliuwk1019) [inline server/backend/api/app/crud/crud_application.py:121]: Hi Kevin, already noted in the meeting.
This probably should be a return of List[models.FamRole] type, and verify if the unit test also pass on this.

File: server/backend/api/app/schemas.py [hunk 2]
```diff
@@ -88,6 +88,13 @@ class FamRoleCreate(BaseModel):
     class Config:
         orm_mode = True
 
+class FamApplicationRoles(FamRoleCreate):
```
Comment (ianliuwk1019) [inline server/backend/api/app/schemas.py:91]: Wondering... you probably meant to use the name: "FamApplicationRole" without "s"?

File: server/backend/tests/crud/test_crud_application.py [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
+import sqlalchemy
 from api.app.crud import crud_application as crud_application
 
 LOGGER = logging.getLogger(__name__)
 
-def test_getFamApplications(dbSession_famApplication_withdata, applicationData1):
-    db = dbSession_famApplication_withdata
+
+def test_getFamApplications(
+    dbSession_famApplication: sqlalchemy.orm.session.Session,
```
Comment (ianliuwk1019) [inline server/backend/tests/crud/test_crud_application.py:14]: Probably can just use "import" as: ( from sqlalchemy.orm import session )

Comment (franTarkenton) [reply server/backend/tests/crud/test_crud_application.py:14]: > Probably can just use "import" as: ( from sqlalchemy.orm import session )

True... I was thinking in the context of defining the type returned by the fixture that being explicit would provide more clarity around what is being returned.

File: server/backend/api/app/schemas.py (hunk 2 above)
Comment (franTarkenton) [reply server/backend/api/app/schemas.py:91]: Yep... That's now resolved in latest push

Comment (franTarkenton) [reply server/backend/api/app/schemas.py]: Resolved this in latest push:

**Review by ianliuwk1019 — APPROVED** [review]

**Pull request merged**